Software-Engineering/L2-Software-Engineer.md

```diff
@@ -0,0 +1,62 @@
+# L2: Software Engineer
+
+> _I am a valuable and enthusiastic member of an Octopus team. I can work independently on small projects, and am focused on learning with a trajectory of growth into the contribution of a Senior Software Engineer or beyond._
+
+- **Planning horizon**: Current and next cycle
+- **Impact radius**: Self plus some team mates
+- **Evaluation**: Manager
+- **Responsibility and direction needed**: Weekly supervision needed from their manager or more senior engineers.
+
+## 🦉 Domain expertise
+
+- I am competent in my domain.
+- I am able to work independently on small projects.
+- I identify and raise potential problems with my peers or lead before getting blocked.
+
+<details>
+<summary>Examples</summary>
+
+- I completed maintenance work in a complex area of the codebase, relying on my knowledge of .NET and occasionally reaching out for help from my more senior teammates when I knew I would need it.
+
+</details>
+
+## 🌱 Teaching and Mentoring
+
+- I use code reviews to strengthen the idioms and practices in use by our team.
```

Comment:
```suggestion
- I use code reviews to strengthen the patterns and practices in use by our team.
```

Suggesting a simpler word in here - some people for whom English isn't their first language might struggle to interpret "idiom".